File: factorial.java
import java.util.Scanner;
public class Main {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int n, i = 1;
		n = in.nextInt();
		
		int fact = 1;
		while(i <= n){
			fact = fact * i;
			i++;
		}
		
		System.out.println(fact);
	}
}

File: factorial.cpp
#include <iostream>
using namespace std;

int main() {
	int n, i = 1;
	cin >> n;

	int fact = 1;
	while(i <= n){
		fact = fact * i;
		i++;
	}

	cout << fact << endl;
	return 0;
}
